### PR TITLE
Uchiwa Support

### PR DESCRIPTION
**Pull request by jbadiapa**
There are a couple of things to change.
- The sensu part (it's only needed because it is a dependency)
- The firewall dependency is not really needed here because is a dependency of sensu/server but it helps to be able to test uchiwa alone.

### CODE REVIEW

File: roles/uchiwa/templates/uchiwa.json.j2 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ {
+   "sensu": [
+     {
+       "name": "{{ uchiwa_sensu_api_server_name }}",
+       "host": "{{ sensu_api_server }}",
+       "port": {{ sensu_api_port }} 
+     }
```
Comment (paramite) [inline roles/uchiwa/templates/uchiwa.json.j2]: Wouldn't it be better to support multiple data centers, eg. have the value of key "sensu" in defaults/main.yml, with the default value of the hash above?

File: roles/uchiwa/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+uchiwa_service_name: uchiwa
+
+uchiwa_bind: 0.0.0.0
+uchiwa_port: 3000
+uchiwa_refresh: 5
+
+uchiwa_file_path: /etc/uchiwa/uchiwa.json
+uchiwa_sensu_api_server_name: uchiwa
+
+datacenters: [{"name": "{{ uchiwa_sensu_api_server_name }}","host": "{{ sensu_api_server }}","port": "{{ sensu_api_port }}" }]
```
Comment (larsks) [inline roles/uchiwa/defaults/main.yml]: This should probably be uchiwa_datacenters, or sensu_datacenters, or something with a name prefix.

**Comment by larsks** [comment]
[Create uchiwa ansible role](https://trello.com/c/5fSsq6Zp/71-create-uchiwa-ansible-role)

**Comment by larsks** [comment]
@jbadiapa, don't merge master into your feature branch; instead `git rebase` your work on top of master:

```
git checkout master
git pull
git checkout my-feature
git rebase master
```

**Comment by jbadiapa** [comment]
I think I've mess up again :(  
So I killed it

**Pull request closed without merging**